infrastructure/application/services/hasura.ts

```diff
@@ -79,12 +79,11 @@ export const createHasuraService = async ({
           ],
           healthCheck: {
             // hasuraProxy health depends on hasura health
-            command: [
-              "CMD-SHELL",
-              `wget --spider --quiet http://localhost:${HASURA_PROXY_PORT}/healthz || exit 1`,
-            ],
-            interval: 15,
-            timeout: 3,
+            // use wget since busybox applet is included in Alpine base image (curl is not)
+            command: ["CMD-SHELL", `wget --spider --quiet http://localhost:${HASURA_PROXY_PORT}/healthz || exit 1`],
+            // generous config; if hasura is saturated/blocking, we give service a chance to scale out before whole task is replaced
```

Comment:
scale out only happens if CPU usage exceeds 30% for 3 minutes in a row (default setup for the pre-defined metrics used below), but if the ALB considers the task unhealthy in advance of that, it will replace it before this can happen!